web/src/app/donate/PayWithPayPalComponent.js

```diff
@@ -51,10 +56,17 @@ class PayWithPayPalComponent extends Component {
         onAuthorize: function(data, actions) {
```

Comment:
Lets move out the anonymous functions into real functions. Just so it is a little easier to maintain

Reply:
talked offline. made the call to make a backlog task for cleaning up this PayPal Component instead of doing the work now